doc/modules/model_persistence.rst

```diff
@@ -5,22 +5,22 @@ Model persistence
 =================
 
 After training a scikit-learn model, it is desirable to have a way to persist
-the model for future use without having to retrain. The following section gives
-you an example of how to persist a model with pickle. We'll also review a few
-security and maintainability issues when working with pickle serialization.
+the model for future use without having to retrain. The following sections give
+you some hints on how to persist a scikit-learn model.
 
-An alternative to pickling is to export the model to another format using one
-of the model export tools listed under :ref:`related_projects`. Unlike
-pickling, once exported you cannot recover the full Scikit-learn estimator
-object, but you can deploy the model for prediction, usually by using tools
-supporting open model interchange formats such as `ONNX <https://onnx.ai/>`_ or
-`PMML <http://dmg.org/pmml/v4-4/GeneralStructure.html>`_.
+.. note::
+
+   Remember that, once exported in a persistent format, the model should only
+   be used for predictions and cannot necessarily be refitted as information
+   about the original hyper-parameters might not have been exported
```

Comment:
this is not true of Pickle as long as the data is stored too.

I feel like at the moment this comment has the potential to raise the confusion more than to avoid errors.

Reply:
I have removed it : the note was already a bit controversial.